<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkDateRangeWidget</class>
 <widget class="QWidget" name="ctkDateRangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>233</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ctkSliderSpinBoxWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="AnyDate">
        <property name="text">
         <string>Any Date</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Today">
        <property name="text">
         <string>Today</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Yesterday">
        <property name="text">
         <string>Yesterday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="LastWeek">
        <property name="text">
         <string>Last Week</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="LastMonth">
        <property name="text">
         <string>Last Month</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SelectRange">
        <property name="text">
         <string>Select Range</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="StartDate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="date">
         <date>
          <year>2010</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>MMM/dd/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="EndDate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="date">
         <date>
          <year>2010</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>MMM/dd/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
